system-variables.md

```diff
@@ -478,6 +478,13 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
     In the case of a poor network environment, appropriately increasing the value of this variable can effectively alleviate error reporting to the application end caused by timeout. If the application end wants to receive the error information more quickly, minimize the value of this variable.
 
+### tidb_batch_insert
+
+- Scope: SESSION
+- Default value: `OFF`
+- This variable permits `tidb_dml_batch_size` to be used in `INSERT` statements.
+- Only the value `OFF` provides ACID compliance. Setting this to any other value will break the atomicity and isolation guarantees of TiDB, as an indivudual `INSERT` statements will be split into smaller transactions.
```

Comment:
```suggestion
- Only the value `OFF` provides ACID compliance. Setting this to any other value breaks the atomicity and isolation guarantees of TiDB, because an individual `INSERT` statement will be split into smaller transactions.
```

Reply:
@leowangpingcap What do you think about this suggestion?